src/foam/nanos/alarming/AlarmNotificationDAO.js

```diff
@@ -85,6 +86,8 @@ foam.CLASS({
       args.put("alarm.started", alarm.getCreated().toString());
       args.put("alarm.cleared", alarm.getIsActive() ? "" : alarm.getLastModified().toString());
       args.put("alarm.note", alarm.getNote());
+      args.put("templateSource", this.getClass().getName());
+      args.put("templateSourceType", EmailTemplateSourceEnum.SERVICE.getLabel());
```

Comment:
So because of the usage of pm on the emailTemplate we don't need EmailTemplateSourceEnum.

The capture on the pmInfo will give us a stack trace if enabled, thus not needing this value to be set or stored.

There was more to this conversation that I need to recall. But removing templateSourceType at the very least seems like a good idea to start with